yarn.lock

```diff
@@ -5192,7 +5192,7 @@ __metadata:
     "@ethersproject/experimental": 5.4.0
     "@ethersproject/providers": 5.4.5
     "@gnosis.pm/safe-apps-web3-react": ^0.6.0
-    "@opendollar/sdk": ^1.5.8-rc.5
+    "@opendollar/sdk": ^1.5.8-rc.4
```

Comment:
Why are we downgrading the SDK version in the yarn.lock but not the package.json?

Reply:
Nice catch, I agree on this one. I think was a local test change that was not supposed to be on this PR. Thanks again @jahabeebs !